clubs_guides/tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+from factory import Faker
+from django.test import TestCase
+
+from clubs_guides.tests.test_factory import ClubsGuideFactory, CategoryFactory
+from clubs_guides.models import Category
+
+
+class TestCategoryListView(TestCase):
+    def setUp(self):
+        self.categories = []
+        for i in range(2):
```

Comment:
lol, yeah usually you use that pattern for modifying an existing dictionary or creating something new based on something existing. Here, we don't have anything existing and we're creating something for the first time so we simulate an existing collection (a list of numbers) to loop over.